### ISSUE TITLE
Enhancement request: hide all code by default

### ISSUE THREAD

**Issue by leeclemmer** [kirbs-/hide_code#30]
Would be great if I could just export/convert notebook to html/pdf and have option to hide all code/prompts globally, instead of having to set which cell to hide cell by cell.

**Comment by kirbs-** [comment]
Yeah, this is on the enhancement list. I'll add it to the next release.

**Comment by leeclemmer** [comment]
Awesome, thanks! Much appreciated :)  Maybe to be a bit more specific: it be great if in the notebook I could see all the code, but when exporting it's all hidden. Thanks!

**Comment by BoPeng** [comment]
Just a quick comment. I have tried to use hide code to generate better HTML output but found two problems (both are feature requests),

1. Hide input and output in HTML, not in Jupyter, because often times we need to perform the analysis with all visible input and output cells, but only want to show certain results in HTML report.
2. Hide all input and most output as default.
3. Allow unhiding hidden input and output content in HTML format. This is hugely helpful because I can have a single report that is suitable for both as a report, and as an archive with all the details of the analysis.

In the end, [we implemented our own `show_output` metadata method](http://vatlab.github.io/SOS/doc/documentation/Notebook_Interface.html) as follows:

1. A keyboard shortcut `Ctrl-Shift-O` to toggle this metadata.
2. A template is provided to output markdown and selected output as default, whiling having a `Show More` button to display all.

The implementation is only limited for [SoS](http://vatlab.github.io/SOS/) though.

Note that we thought of adding the features as an extension to hide code but had to give up because hide code does not support nbconvert 5.0+.

**Comment by kirbs-** [comment]
Thanks for the feedback! I'm going to get around to the enhancements some time this summer.

**Comment by alanamarzoev** [comment]
Hi! What's the timeline like for the release of this enhancement?

**Comment by kirbs-** [comment]
@alanamarzoev Should have this done in the next 4-5 weeks.
